packages/web3-eth/src/errors.ts

```diff
@@ -265,13 +266,26 @@ export class TransactionDataAndInputError extends InvalidValueError {
 	}
 }
 
+export class TransactionSendTimeoutError extends InvalidValueError {
+	public code = ERR_TX_SEND_TIMEOUT;
+
+	public constructor(value: { numberOfSeconds: number; transactionHash?: Bytes }) {
+		super(
+			`transactionHash: ${
+				value.transactionHash ? value.transactionHash.toString() : 'not available'
+			}`,
+			`The connected Ethereum Node did not respond within ${value.numberOfSeconds} seconds, please make sure your transaction was properly sent and you are connected to a healthy Node. Be aware that it might still be mined! We recommend checking the pending transactions for such a case.`,
+		);
+	}
+}
+
 export class TransactionPollingTimeoutError extends InvalidValueError {
 	public code = ERR_TX_POLLING_TIMEOUT;
 
 	public constructor(value: { numberOfSeconds: number; transactionHash: Bytes }) {
 		super(
 			`transactionHash: ${value.transactionHash.toString()}`,
-			`Transaction was not mined within ${value.numberOfSeconds} seconds, please make sure your transaction was properly sent. Be aware that it might still be mined!`,
+			`Transaction was not mined within ${value.numberOfSeconds} seconds, please make sure your transaction was properly sent. Be aware that it might still be mined!  We recommend checking the pending transactions for such a case.`,
```

Comment:
```suggestion
			`Transaction was not mined within ${value.numberOfSeconds} seconds, please make sure your transaction was properly sent. Be aware that it might still be pending or mined!`,
```